src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -31,33 +31,35 @@ packages = [
     { include = "ansys", from = "src" },
 ]
 
+[[tool.poetry.source]]
+name = "PyPI"
+priority = "primary"
+
 [[tool.poetry.source]]
 name = "solutions-private-pypi"
 url = "https://pkgs.dev.azure.com/pyansys/_packaging/ansys-solutions/pypi/simple/"
 priority = "supplemental"
 
-[[tool.poetry.source]]
-name = "PyPI"
-priority = "primary"
-
 [tool.poetry.dependencies]
 python = ">=3.8, <3.11"
-ansys-dash-treeview = {version = "0.0.1.dev0", source = "solutions-private-pypi"}
 ansys-saf-portal = {version = "1.0rc3", source = "solutions-private-pypi"}
-ansys-saf-desktop = {version = "1.0rc3", source = "solutions-private-pypi"}
+ansys-saf-desktop = {version = "1.0rc4", source = "solutions-private-pypi"}
 ansys-solutions-dash-lib = {version = "^0.4", source = "solutions-private-pypi"}
```

Comment:
Shouldn't this and `ansys-web-components-dash` be included in the conditional on line 49 below?